p2pserver/message/types/version.go

```diff
@@ -31,13 +31,14 @@ type VersionPayload struct {
 	TimeStamp    int64
 	SyncPort     uint16
 	HttpInfoPort uint16
-	ConsPort     uint16
-	Cap          [32]byte
-	Nonce        uint64
-	StartHeight  uint64
-	Relay        uint8
-	IsConsensus  bool
-	SoftVersion  string
+	//TODO
+	ConsPort    uint16
```

Comment:
remove ConsPort and IsConsensus ?

Reply:
change packet format is incompatible, I suggest adding `TODO` document to remind us to remove those when we have the chance to upgrade the network layer.